components/omega/doc/design/OmegaV1GoverningEqns.md

```diff
@@ -0,0 +1,749 @@
+= D^\varphi + Q^\varphi
+$$ (continuous-tracer)
+
+Here we have express the following terms as a general operators, with examples of specific forms provided below: the dissipation ${\bf D}^u$, momentum forcing ${\bf F}^u$, tracer diffusion $D^\varphi$, and tracer sources and sinks $Q^\varphi$. The momentum equation can be rewritten using the product rule on $\rho {\bf u}$, mass conservation, and dividing by $\rho$, as:
+
+$$
+\frac{D {\bf u}_{3D} }{D t} \equiv
+\frac{\partial {\bf u}_{3D}}{\partial t}
+ + \nabla_{3D} \cdot \left( {\bf u}_{3D} \otimes {\bf u}_{3D}  \right)
+  = - \frac{1}{\rho} \nabla_{3D} p
++ {\bf D}^u + {\bf F}^u
+$$ (continuous-momentum-rho)
+
+Geophysical fluids such as the ocean and atmosphere are rotating and stratified, and horizontal velocities are orders of magnitude larger than vertical velocities. It is therefore convenient to separate the horizontal and vertical as ${\bf u}_{3D} = \left( {\bf u}, w \right)$ and $\nabla_{3D} = \left( \nabla_z, d/dz \right)$ where $z$ is the vertical direction in a local coordinate frame on the Earth's surface and $w$ is the vertical velocity. The $z$ subscript on $\nabla_z$ is to remind us that this is the true horizontal gradient (perpendicular to $z$), as opposed to gradients within tilted layers used in the following section. The gravitational force $g {\bf k}$ is included in ${\bf F}^u$. The rotating frame of reference results in the Coriolis force $f {\bf k} \times {\bf u} \equiv f {\bf u}^\perp$, where $f$ is the Coriolis parameter and ${\bf u}^\perp$ is the horizontal velocity rotated $90^o$ from $\bf u$. See any textbook in the [References](#references) for a full derivation.
+
+The horizontal parts of the advective term may be replaced with the vector identity
+
+$$
+\begin{aligned}
+\nabla_z \cdot \left( {\bf u} \otimes {\bf u}  \right) &=
+\boldsymbol{u} \cdot \nabla_z \boldsymbol{u} \\
+&= (\nabla_z \times \boldsymbol{u}) \times \boldsymbol{u} + \nabla_z \frac{|\boldsymbol{u}|^2}{2} \\
+&= \{\boldsymbol{k} \cdot (\nabla_z \times \boldsymbol{u})\} \boldsymbol{k} \times \boldsymbol{u} + \nabla_z \frac{|\boldsymbol{u}|^2}{2} \\
+&= \zeta \boldsymbol{u}^{\perp} + \nabla_z K.
+\end{aligned}
+$$ (advection-identity)
+
+where $\zeta$ is relative vorticity and $K$ is kinetic energy. This step separates the horizontal advection into non-divergent and non-rotational components, which is useful in the final TRiSK formulation.
+
+#### Assumptions
+
+For a primitive equation ocean model, we assume the fluid is incompressible and hydrostatic. For Omega we are not making the Boussinesq assumption, so all density terms are the full density. In particular, the density coefficient of the pressure gradient in [](#continuous-momentum) is not a constant, as it is in primitive equation models like POP and MPAS-Ocean.
+
+**Incompressible:** Assume that the three-dimensional velocity field is nondivergent,
+
+$$
+ \nabla_z \cdot {\bf u} + \frac{\partial w}{\partial z}  = 0.
```

Comment:
This implies volume conservation.

Reply:
That is an excellent point. From [source](https://www.sciencedirect.com/topics/engineering/incompressible-fluid): "An incompressible fluid has zero compressibility; therefore, regardless of pressure, it has a constant density".  It sounds like a non-Boussinesq fluid is by definition not incompressible. But a compressible fluid supports sound waves, and we definitely don't want that. That is confusing. The [MIT-GCM equation set](https://mitgcm.readthedocs.io/en/latest/overview/eqn_motion_ocn.html) might help.

Reply:
My research suggests that non-Boussinesq models use the full mass-conservation equation:
```suggestion
 \frac{\partial \rho}{\partial t} + \nabla \cdot (\rho \mathbf{u}) = 0
```
My understanding is that sound waves get filtered out because we're using hydrostatic balance for the vertical momentum equation, but I also see some indications that other filtering techniques may be needed.  See:
https://acmeclimate.slack.com/archives/C08PQP0U64R/p1746367303787279

Reply:
I added a paragraph on acoustic waves and why we are either filtering them out (hydrostatic approximation) or should be able to resolve them in the barotropic substepping (in the case of Lamb waves).